CLDR-15141 add rohingya lc

stenshamn opened the pull request:
CLDR-15141

- [X] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see http://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-15141)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: http://www.unicode.org/copyright.html#License
-->

srl295 commented:
@stenshamn the commit message and PR needs to start with `CLDR-15141` … no space after CLDR.. I fixed the PR but cant fix the commit

srl295 commented on tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/Locales.txt:
```diff
@@ -886,6 +886,8 @@ Mozilla	;	und	;	modern
 
 Rodakych ; pcm ; modern ; Nigerian Pidgin
 
+rohingyazuban	;	rhg	;	modern
```
LGTM but do we need to have the script? are they covering rhg_Arab and rhg_Rohg ?

stenshamn replied:
I figured by leaving it unstated they would be able to work with both?

pedberg-icu replied:
The default script is Rohg, so unspecified implies that I think.

stenshamn commented:
> @stenshamn the commit message and PR needs to start with `CLDR-15141` … no space after CLDR.. I fixed the PR but cant fix the commit

I caught that, but thought I fixed it in when I squashed the commits. That didn't stick?
